package.json

```diff
@@ -1,10 +1,18 @@
 {
   "name": "@politico/vue-accessible-selects",
-  "version": "0.0.1",
+  "version": "0.0.2-alpha.5",
   "description": "Select & Multi Select implementations for Vue, focused especially on implementing accessibility best practices",
-  "main": "index.js",
+  "files": [
+    "dist"
+  ],
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "sideEffects": false,
   "scripts": {
-    "test": "echo \"write some tests\""
+    "test": "echo \"write some tests\"",
+    "build": "NODE_ENV=production rollup --config rollup.config.js && rm -rf dist/styles && cp -r src/styles dist/styles",
```

Comment:
If you're going to use the default config file and location of it, you can simply pass `-c` to rollup and avoid the unnecessary verbosity. https://rollupjs.org/guide/en/#using-config-files

Also, why are you replacing the compiled styles?

Reply:
Ah, neat! Will try...

And a great question on the styles. For now, I wanted to export the SCSS file directly, and Rollup isn't actually exporting any styles right now (the `rm -rf dist/styles` is just to ensure cleaning up the old ones)

But that's actually part of a conversation we'll be having today: how do we want to approach styles for these types of packages, and does that mean exporting both `.scss` and compiled `.css`, for example?